lib/elixir_analyzer/test_suite/newsletter.ex

```diff
@@ -107,6 +139,14 @@ defmodule ElixirAnalyzer.TestSuite.Newsletter do
         end
       end
     end
+
+    form do
+      def open_log(_ignore) when _ignore do
+        _block_includes do
+          File.open!(_ignore, [:write])
```

Comment:
Let's also include all of the variations we had before (with and without `File`, with and without `!`). I know they are not all super likely, but we might as well.

Reply:
I didn't remove any of the existing variants though.

Reply:
I meant let's duplicate all four existing variants and add the `when _ignore`, not just for `File.open!`.

Right now if someone uses 
```elixir
        import File
        def open_log(path) when is_binary(path) do
          open!(path, [:write])
        end
```
It will trigger a comment.

Reply:
Gotcha!